src/rdpc/query/fetchDonorIdsByAnalysis.ts

```diff
@@ -34,33 +31,28 @@ type QueryResponseData = {
 
 type QueryVariable = {
   analysisId: string;
-  analysisState: string;
 };
 
 const fetchDonorIdsByAnalysis = async ({
   analysisId,
-  action,
   rdpcUrl,
   egoJwtManager,
 }: {
   analysisId: string;
-  action: string;
   rdpcUrl: string;
   egoJwtManager: EgoJwtManager;
 }) => {
   const jwt = (await egoJwtManager.getLatestJwt()) as EgoAccessToken;
   const accessToken = jwt.access_token;
   return await promiseRetry<string[]>(async (retry) => {
     try {
-      const analysisState = determineAnalysisState(action);
       const body = JSON.stringify({
         query,
         variables: {
           analysisId,
-          analysisState,
         } as QueryVariable,
       });
-      logger.info(`RDPC API Request body: ${body}`);
+      logger.info(`fetchDonorIdsByAnalysis: RDPC API Request body: ${body}`);
```

Comment:
This was just temporary, for troubleshooting this issue. Can you remove this for me?

Reply:
Can you also remove the `logger.info()` call in the then block that is outputting the analysis?